examples/inference.py

```diff
@@ -126,7 +117,49 @@ def main(
             if not is_safe:
                 print(method)
                 print(report)
-                
+    return output_text
+
+  if prompt_file is not None:
+      assert os.path.exists(
+          prompt_file
+      ), f"Provided Prompt file does not exist {prompt_file}"
+      with open(prompt_file, "r") as f:
+          user_prompt = "\n".join(f.readlines())
+      evaluate(user_prompt, temperature, top_p, top_k, max_new_tokens)
+  elif not sys.stdin.isatty():
```

Comment:
We need this part because we can pass --prompt_file parameter as follows:
```
python examples/inference.py --model_name '../models/Llama-2-7b-hf' --peft_model 'outputs/7b' --max_new_tokens 580  --quantization true  --prompt_file examples/samsum_prompt.txt
```

Reply:
What I mean is passing prompt file is being handled in lines 122-128, then if we dont pass the --prompt_file it straight goes to gradio. I believe lines 129-131 are not needed here.

Reply:
```
cat examples/samsum_prompt.txt |  python examples/inference.py --model_name '../models/Llama-2-7b-hf' --peft_model 'outputs/7b' --max_new_tokens 580  --quantization true
```
If we run inference.py commnad like the above, 129-131 lines are necessary.